bundled/tool/server.py

```diff
@@ -240,7 +246,7 @@ def code_action(params: lsp.CodeActionParams) -> List[lsp.CodeAction]:
     return code_actions
 
 
-@QUICK_FIXES.quick_fix(codes=["C0301:line-too-long"])
+@QUICK_FIXES.quick_fix(codes=["C0301:line-too-long", "C0305:trailing-newlines"])
```

Comment:
Thank you for the PR! I think this change (line 249) and the one in the test_code_actions.py are the only ones that are needed for the issue in question. Would you mind reverting the other changes? You can do it by running `git revert 732594c8dc6d4e93d64a76ede184d6fc291aef90` -- which will revert your latest commit. Then you can add the test again, commit, and push your changes to this PR again. Does that make sense?

Reply:
Ok it should be good to go now! Let me know if any other changes need to be made. Thanks for the review